.github/workflows/marketplace-publish.yml

```diff
@@ -38,7 +38,10 @@ jobs:
           npm run vsix
           package=$(unzip -l bin/roo-cline-${current_package_version}.vsix)
           echo "$package"
-          echo "$package" | grep -q "dist/extension.js" || exit 1
+          echo "$package" | grep -q "extension/dist/package.json" || exit 1
+          echo "$package" | grep -q "extension/dist/package.nls.json" || exit 1
+          echo "$package" | grep -q "extension/dist/extension.js" || exit 1
+          echo "$package" | grep -q "extension/webview-ui/audio/celebration.wav" || exit 1
```

Comment:
Additional checks to ensure we're building the vsix correctly. We recently changed the location of the `audio` directory. I'd like to come up with a better solution for ensuring certain build artifacts are always present. You can end up breaking things if you add something without realizing that `.vscodeignore` disallows it.